common/lib/capa/capa/templates/status_icon.html

```diff
@@ -0,0 +1,10 @@
+<%page expression_filter="h" args="status"/>
+<%! from django.utils.translation import ugettext as _ %>
+
+% if status == 'correct':
+  <span class="status-icon ${status.classname} fa fa-check"></span>
```

Comment:
I meant to use the existing @include status-icon mixin.
.status-icon {
  &.correct {
    @include status-icon($correct, $checkmark-icon);
  }
 &.unsubmitted, &.unanswered {
   content: '';
 }
...
}